### PR TITLE
Closes #102 ophtha sc@devel

### LINKED ISSUE #102: Create SC for admiralophtha
See code: https://github.com/pharmaverse/admiralophtha/blob/devel/dev/sc.R

@bms63 Should SDTM.SC be called admiral_sc or admiralophtha_sc as it is Ophtha specific.

### PR DESCRIPTION
**Pull request by millerg23**
Thank you for your Pull Request! 

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. 

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: dev/sc.R [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+dm <- admiral_dm
+sv <- admiral_sv
+
+# Remove screen failures, they will not make it to drug infusion
+dm1 <- dm %>%
+  filter(ARMCD != "Scrnfail")
+
+
+# use subjects in DM  and info from SV  Screening 1 visit
```
Comment (bms63) [inline dev/sc.R:18]:
```suggestion
# Use subjects in DM  and info from SV  Screening 1 visit
```

File: dev/sc.R [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    "STUDYID", "DOMAIN", "USUBJID", "SCSEQ", "SCTESTCD", "SCTEST",
+    "SCCAT", "SCORRES", "SCSTRESC", "EPOCH", "SCDTC", "SCDY"
+  )
+
+sc <- sc_seq %>%
+  ungroup() %>%
+  # sort data
+  arrange(STUDYID, USUBJID, SCSEQ)
+
+# assign variable labels
```
Comment (bms63) [inline dev/sc.R:51]:
```suggestion
  # Sort data
  arrange(STUDYID, USUBJID, SCSEQ)

# Assign variable labels
```

File: dev/sc.R [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+attr(sc$STUDYID, "label") <- "Study Identifier"
+attr(sc$DOMAIN, "label") <- "Domain Abbreviation"
+attr(sc$USUBJID, "label") <- "Unique Subject Identifier"
+attr(sc$SCSEQ, "label") <- "Sequence Number"
+attr(sc$SCTESTCD, "label") <- "Subject Characteristic Short Name"
+attr(sc$SCTEST, "label") <- "Subject Characteristic"
+attr(sc$SCCAT, "label") <- "Category for Subject Characteristic"
+attr(sc$SCORRES, "label") <- "Result or Finding in Original Units"
```
Comment (bms63) [inline dev/sc.R:59]: Just checking that these are correct?

File: dev/sc.R [hunk 4]
```diff
@@ -0,0 +1,71 @@
+# SC (ophthalmology)
+
+library(admiral)
```
Comment (bms63) [inline dev/sc.R:3]: Is admiral needed here?

File: dev/sc.R [hunk 5]
```diff
@@ -0,0 +1,71 @@
+# SC (ophthalmology)
```
Comment (bms63) [inline dev/sc.R:1]: Could we reference the pkgdown site - just for future devs coming to look at this script and wondering what it is for?

File: dev/sc.R (hunk 1 above)
Comment (millerg23) [reply dev/sc.R]: Fixed

File: dev/sc.R (hunk 2 above)
Comment (millerg23) [reply dev/sc.R]: Fixed

File: dev/sc.R (hunk 3 above)
Comment (millerg23) [reply dev/sc.R]: These 4 look fine to me.

File: dev/sc.R (hunk 4 above)
Comment (millerg23) [reply dev/sc.R]: dont think so, have removed.

File: dev/sc.R (hunk 5 above)
Comment (millerg23) [reply dev/sc.R:1]: there is work being done on the site, for the upcoming release. did we do this for `onco`, should we maybe have a page with the different admiral package sites where `admiral.test` is being used?

**Comment by millerg23** [comment]
BTW I used metatools to create labels to try and be consistent.

File: dev/sc.R (hunk 5 above)
Comment (bms63) [reply dev/sc.R:1]: not a standard practice - but I would like to start pointing to sites/repos of where this data is being used.  Just to let people a few years down the road know about where the data is being utilized.

If not available - maybe we could just point to the repo.

Comment (bms63) [reply dev/sc.R:1]: We could put a blurb in the news that admiral.test has sdtm data for oncology and opthat available and link to pkgdown sites.  I'll make an issue to do this before the release.

**Comment by millerg23** [comment]
Let me know if I need to make any more updates. Thanks!

**Review by bms63 — APPROVED** [review]

**Pull request merged**